src/Graphics/Image/Processing/Gabor.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE RankNTypes #-}
```

Comment:
I suppose this file should not be part of this PR but it just slipped in?

Reply:
Yes sir, very sorry. Will correct it.